module/bdev/lvol/vbdev_lvol.c

```diff
@@ -1379,3 +1379,74 @@ vbdev_lvol_create_with_uuid(struct spdk_lvol_store *lvs, const char *name, uint6
 
 	return rc;
 }
+
+int
+vbdev_lvs_create_with_uuid(const char *base_bdev_name, const char *name, const char *uuid,
```

Comment:
what about having the original create function call this one, with the uuid set to `NULL` to avoid code dup?